src/syscall/syscall_libc.go

```diff
@@ -270,6 +280,10 @@ func libc_mkdir(pathname *byte, mode uint32) int32
 //export rmdir
 func libc_rmdir(pathname *byte) int32
 
+// int rename(const char *from, *to);
+//export rename
+func libc_rename(from, too *byte) int32
```

Comment:
typo for `too` or intentional?

Reply:
Typo, but let's leave it out of superstition :-) and because I'm getting tired of clicking "retry all builds" to work around the spurious CI problems...